app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  def markdown(text)
```

Comment:
Docs?

I'm not inclined to be picky about controllers, but I think a docstring on helper methods is a good idea.